requirements.in

```diff
@@ -7,7 +7,7 @@ django-ordered-model
 django-phonenumber-field
 django-widget-tweaks
 phonenumberslite
-psycopg2
+psycopg2>=2.9.3
```

Comment:
This is not required. `pip-compile -P psycopg2 requirements.in` would have the same effect.